docs/user/recipes/request-id.rst

```diff
@@ -10,14 +10,14 @@ to every log entry.
 
 If you wish to trace each request throughout your application, including
 from within components that are deeply nested or otherwise live outside of the
-normal request context, you can use a `thread-local`_ context object to store
+normal request context, you can use a `contextvars`_ object to store
```

Comment:
We could simply replace this with an Intersphinx link to the `contextvars` module, but this applies to the current version too, so not a requirement for merging this.